app/filevalueresolver.go

```diff
@@ -0,0 +1,74 @@
+package app
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/TIBCOSoftware/flogo-lib/config"
+	"github.com/TIBCOSoftware/flogo-lib/logger"
+)
+
+var preload = make(map[string]interface{})
+
+var log = logger.GetLogger("app-props-resolver-file")
+
+// Comma separated list of json files overriding default application property values
+// e.g. FLOGO_APP_PROPS_FILE_CONFIG=app1.json,common.json
+const EnvAppPropertyFileConfigKey = "FLOGO_APP_PROPS_FILE_CONFIG"
+
+func init() {
+
+	filePaths := getExternalFiles()
+	if filePaths != "" {
+		// Register value resolver
+		RegisterPropertyValueResolver("file", &FileValueResolver{})
+
+		if config.GetAppPropertiesValueResolver() == "" {
+			//Make file resolver default since FLOGO_APP_PROPS_FILE_CONFIG is set
```

Comment:
I don't understand why the fact that the environment variable is set implies that the resolver becomes the default property resolver. Will it be the same for other resolvers ? How the priorities will be managed ?

Reply:
@debovema : This is for the developer's convenience. Note that I am setting it implicitly only when _FLOGO_APP_PROPS_VALUE_RESOLVER_ is not explicitly set. I dont think we should trouble developers for simple configuration like value resolver when intention is to use only one resolver like file or consul or env.
If developers intend to use multiple of those (chaining), then they should explicitly set _FLOGO_APP_PROPS_VALUE_RESOLVER_.

Reply:
🤔 I kind of tend to lean toward not setting a default... I think it may make more sense for them to explicitly set the resolver to `file` not just assume that's what they want if file name(s) have been specified in the other env var... Could make troubleshooting a bit more of a challenge as we're inferring behavior which may not be what is expected... I'd rather stay away from this. If the file resolver isn't set, regardless of them specifying files in another env var, then the default resolver is used (or none at all)...

Reply:
Alright. Will remove it for now.